config/locales/eng/models.yml

```diff
@@ -32,6 +32,8 @@ eng:
         started_on: "start-up date"
       fixed_asset_depreciation:
         asset_amount: "Accountable value"
+      journal_entry_item:
+        name: "Booking text"
```

Comment:
Has this "Booking text" been specified by a spec or is this a loose translation ?